yb-voyager/src/mat/scripts/pgPsql.sql

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
I guess, we don't need this file anymore, right? There is one in the `matscripts`?

Reply:
yes, removing it